loleaflet/src/layer/tile/CanvasTileLayer.js

```diff
@@ -349,10 +351,6 @@ L.CanvasTilePainter = L.Class.extend({
 		var partChanged = (part !== this._lastPart);
 
 		var mapSizeChanged = !newMapSize.equals(this._lastMapSize);
-		// To avoid flicker, only resize the canvas element if width or height of the map increases.
-		var newSize = new L.Point(Math.max(newMapSize.x, this._lastSize ? this._lastSize.x : 0),
-					  Math.max(newMapSize.y, this._lastSize ? this._lastSize.y : 0));
-		var resizeCanvas = !this._lastSize || !newSize.equals(this._lastSize);
```

Comment:
If I remember correctly this was used to avoid flicker when changing to/from cell edit-mode in a device with on-screen keyboard and also during sidebar switch in desktop browser. It would be good to check for this behavior.

Reply:
I checked them now, they seem good.